docs/source/notes/libtorch_stable_abi.md

```diff
@@ -128,3 +128,26 @@ The above is relevant in two places:
     ```cpp
     aoti_torch_call_dispatcher(const char* opName, const char* overloadName, StableIValue* stack)
     ```
+
+### Versioning and Forward/Backward compatibility guarantees
+
+We provide a `TORCH_ABI_VERSION` macro in `torch/headeronly/version.h` of the form
+
+```
+[ byte ][ byte ][ byte ][ byte ][ byte ][ byte ][ byte ][ byte ]
+[MAJ   ][ MIN  ][PATCH ][                 ABI TAG              ]
+```
+
+In the present phase of development, APIs in the C-shim will be versioned based on major.minor.patch release that they are first introduced in, with 2.10 being the first release where this will be enforced. The ABI tag is reserved for future use.
+
+Extensions can select the minimum abi version to be compatible with using:
+
+```
+#define TORCH_TARGET_VERSION …
```

Comment:
let's also give folks an example of what it could be (like the raw numbers) so they can copy/paste and adjust